one_fm/hiring/doctype/work_contract/work_contract.py

```diff
@@ -41,12 +41,14 @@ def update_on_workflow_state(self):
 		if self.workflow_state == 'Send to Authorised Signatory':
 			self.validate_attachments()
 			self.validate_authorized_signatory()
-			email_authority_for_signature(self)
+			#email_authority_for_signature(self)
```

Comment:
If this line is not needed, you can remove it instead of commenting it

Reply:
@SalShai What is one_fm/hiring/notification/__init__.py

Reply:
I created a Notification List for Work Contract to sent Notifiction for Approval.

Reply:
I used this method because Workflow didn't help.

Reply:
Okay